README.md

```diff
@@ -25,7 +25,7 @@ Deepin movie is Deepin Desktop Environment Movie Player.
 
 2. Build:
 ```
-$ cd deepin-viewer
+$ cd deepin-movie-reborn(tab)
```

Comment:
If user fetched the source from GitHub (via `git clone`), they will probably not got any extra version number, so please remove the *(tab)*

```suggestion
$ cd deepin-movie-reborn
```